src/Jh/DataImportMagento/Writer/ShipmentWriter.php

```diff
@@ -64,6 +70,17 @@ public function writeItem(array $item)
                 ->addObject($shipment->getOrder())
                 ->save();
 
+            if (isset($item['tracking_number'], $item['carrier'])) {
+                $tracking = clone $this->trackingModel;
+                $tracking
+                    ->setShipment($shipment)
+                    ->setData('title', $item['carrier'])
+                    ->setData('number', $item['tracking_number'])
+                    ->setData('carrier_code', 'custom')
+                    ->setData('order_id', $shipment->getData('order_id'))
+                    ->save();
+                \Mage::dispatchEvent('after_shipping_data_save', ['item' => $item]);
```

Comment:
Thats Radials Import Observer, and its listening to the `after_shipping_data_save` event that I've now removed, Thats how I previously got access to $item.